specification/use_case_library.md

```diff
@@ -36,7 +36,7 @@ The following table contains a set of commonly performed FinOps scenarios that w
 | FinOps Practitioner | Managing Anomalies | As a FinOps Practitioner, I need to see the daily costs across all cloud providers, billing accounts, sub accounts, and region | BillingAccountId<br>SubAccountId<br>ChargePeriodStart<br>ChargePeriodEnd<br>EffectiveCost<br>Provider<br>Region |
 | FinOps Practitioner | Managing Anomalies | As a FinOps practitioner, I need to see the daily costs across all cloud providers, billing accounts, sub accounts, and service | BillingAccountId<br>SubAccountId<br>ChargePeriodStart<br>ChargePeriodEnd<br>EffectiveCost<br>Provider<br>ServiceName |
 | FinOps Practitioner | Managing Commitment Based Discounts | As a FinOps Practitioner, I want to track all commitment based discounts purchased for a time period | Provider<br>BillingAccountID<br>CommitmentDiscountId<br>CommitmentDiscountType<br>BilledCost<br>ChargePeriodStart<br>ChargeCategory |
-| FinOps Practitioner | Managing Commitment Based Discounts | As a FinOps Practitioner, I want to track unused commitment charges in any given time period so that I consider them in my future commitment planning or remedy them | ChargeSubcategory (filter)<br>CommitmentDiscountID<br>BilledCost<br>ChargePeriodStart |
+| FinOps Practitioner | Managing Commitment Based Discounts | As a FinOps Practitioner, I want to track unused commitment charges in any given time period so that I consider them in my future commitment planning or remedy them | CommitmentDiscountUsage (filter)<br>CommitmentDiscountID<br>BilledCost<br>ChargePeriodStart |
```

Comment:
```suggestion
| FinOps Practitioner | Managing Commitment Based Discounts | As a FinOps Practitioner, I want to track unused commitment charges in any given time period so that I consider them in my future commitment planning or remedy them | CommitmentDiscountStatus (filter)<br>CommitmentDiscountID<br>BilledCost<br>ChargePeriodStart |
```